packages/wallets/package.json

```diff
@@ -28,12 +28,14 @@
         "react-router-dom": "^5.2.0",
         "react-transition-group": "4.4.2",
         "usehooks-ts": "^2.7.0",
-        "yup": "^0.32.11"
+        "yup": "^0.32.11",
```

Comment:
Is this how we are planning on validating schemas? I thought [zod](https://zod.dev/) was the preference? @yashim-deriv

Reply:
Initially I had suggested to use Zod, but Formik does not work with Zod at the moment but is compatible with Yup. There is an adapter to resolve this but there are some functions that are not supported.

Reply:
The team preferred using `yup` instead of `zod` because it works well with `Formik`. I think @adrienne-deriv can clarify more on this.